### PR TITLE
[EC-255 ] Create update authorization button on the dashboard

### PR DESCRIPTION
**Pull request by MartinM1312**
## 💅🏼 What issue does this fix?
This PR creates an update authorization button on the dashboard

## 🍂 Before You Submit

* [ ] Did you write & run tests and linters?
* [ ] Did you run Google Lighthouse and/or WebAIM (Wave) on UI components in your PR?
* [ ] Does your PR contain any required translations?
* [ ] Are your primary keys UUIDs on any new tables?

### CODE REVIEW

File: client/src/Dashboard/DashboardTable.js [hunk 1]
```diff
@@ -30,11 +30,15 @@ export default function DashboardTable({
...
   const [isMIModalVisible, setIsMIModalVisible] = useState(false)
+  const [isAUModalVisible, setIsAUModalVisible] = useState(false)
   const [selectedChild, setSelectedChild] = useState({})
+  const [effectiveDate, setEffectiveDate] = useState(null)
+  const [expirationDate, setExpirationDate] = useState(null)
   const [inactiveDate, setInactiveDate] = useState(null)
   const [activeDate, setActiveDate] = useState(null)
   const [inactiveReason, setInactiveReason] = useState(null)
   const [sortedRows, setSortedRows] = useState([])
+  const [auErrorMessage, setAUErrorMessage] = useState('')
```
Comment (fabianraf) [inline client/src/Dashboard/DashboardTable.js:41]: is this a typo? setAUE (AU in capital letters)

Comment (MartinM1312) [reply client/src/Dashboard/DashboardTable.js:41]: Is meant for Authorization Update, maybe changing auErrorMessage to aUErrorMessage would be more consistent?

File: client/src/Dashboard/DashboardTable.js [hunk 2]
```diff
@@ -254,25 +258,92 @@ export default function DashboardTable({
...
+        <>
+          <p className="text-lg text-gray10">{full_name}</p>
+          <p className="text-base text-gray8">Current Effective Date:</p>
+          <p>{effective_date}</p>
+          <p className="text-base text-gray8">Current Expiration Date:</p>
+          <p>{expiration_date}</p>
+          <br></br>
+          <p className="text-base text-gray10">New Effective Date</p>
+          <DatePicker
+            style={{
```
Comment (fabianraf) [inline client/src/Dashboard/DashboardTable.js:279]: can't these styles be added as a class?

**Review by fabianraf — DISMISSED** [review]

File: client/src/Dashboard/DashboardTable.js (hunk 1 above)
Comment (fabianraf) [reply client/src/Dashboard/DashboardTable.js:41]: yes, that makes sense

**Review by fabianraf — APPROVED** [review]

**Pull request merged**